1st negative number in every window size of k

bitaashna opened the pull request:
#2304
Description

One of the fixed window problem has been solved and implemented using c++.

Checklist

I've read the contribution guidelines.
I've checked the issue list before deciding what to submit.
I've edited the README.md and link to my code.
#C++
#2304

bitaashna committed:
sliding_window

bitaashna committed:
sliding_window

bitaashna committed:
sliding_window_1

bitaashna commented:
@siddharth25pandey Now also facing the same issue.
One of the checks is failing.
DeepSource: Java (beta)
what should i do?

bitaashna committed:
sliding_window_1

siddharth25pandey commented:
> @siddharth25pandey Now also facing the same issue.
> One of the checks is failing.
> DeepSource: Java (beta)
> what should i do?

@bitaashna Ignore it, no need to worry about as Deepsource Presently don't supports C++

siddharth25pandey commented on C-Plus-Plus/README.md:
```diff
@@ -236,6 +236,16 @@ _add list here_
+
+- [Maximum sum subarray of size k](sliding_window/max_sum_subarray_of_size_k.cpp)
+- [1st negative number in every window size of k](sliding_window/1st_negative_no_in_every_window_of_size_k.cpp)
+- [Count occurence of Anagram](sliding_window/count_occurences_of_anagrams.cpp)
+- [Maximum of all subarray of size k](sliding_window/maximum_of_all_subarray_of_size_k.cpp)
+- [Largest subarray of sum k](sliding_window/largest_subarray_of_sum_k.cpp)
+- [largest substring with k distinct character](sliding_window/longest_substring_with_k_unique_characters.cpp)
+- [length of largest substring with no repeating characters](sliding_window/length_of_longest_substring_with_no_repeating_characters.cpp)
```
Remove this

bitaashna replied:
I have removed the file.
But in readme I have updated all at once.
Should I do it again?
Please let me know

siddharth25pandey commented on C-Plus-Plus/README.md:
```diff
@@ -236,6 +236,16 @@ _add list here_
 
+## Sliding Window
+
+- [Maximum sum subarray of size k](sliding_window/max_sum_subarray_of_size_k.cpp)
```
Remove this

siddharth25pandey reviewed (changes requested):
@bitaashna as told you earlier, please create one PR per program
* Remove the count_of_occurences_of_anagrams file

bitaashna committed:
Delete C-Plus-Plus/sliding_window directory

bitaashna commented:
@siddharth25pandey  I have updated the readme file at once but i have added files one by one by making different branches in forked repository.
![Screenshot from 2021-03-15 17-55-04](https://user-images.githubusercontent.com/57864526/111153183-a34a5880-85b7-11eb-9f8e-286728c72ce6.png)
To uncommit the changes I deleted my repo :( from main3 branch. Now I am not able to commit it again.Please help me regarding this.

siddharth25pandey commented:
@bitaashna Duplicate [PR](https://github.com/TesseractCoding/NeoAlgo/pull/3277)
